lost/src/main/java/com/mapzen/android/lost/internal/FusionEngine.java

```diff
@@ -112,6 +118,10 @@ public FusionEngine(Context context, Callback callback) {
     if (passiveInterval < Long.MAX_VALUE) {
       enablePassive(passiveInterval);
     }
+
+    if (prepLooper) {
+      Looper.loop();
```

Comment:
@amayatsky Lost does indeed use the `LocationManager` under the hood. The `FusionEngine` is wrapped in a started service and registered for updates as a `LocationListener`.

If we instead wrap the `FusionEngine` in an `IntentService` and register it with the `LocationManager` via `PendingIntent` this could potentially help solve our issue of background use cases when the app is stopped.